CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Adds
 
 * Adds regex pattern feature for string fields.
+* Adds `pnpm` support. Be sure to install with peer dependencies - `pnpm install --config.auto-install-peers=true`. Introduces new Apostrophe root configuration `pnpm` to force opt-in/out when auto detection fails (usually caused by monorepo tools like `rush`). 
```

Comment:
Link to a new page in the docs instead for details, so you can spell out specifics clearly, like the fact that v7 and v8 of pnpm have different attitudes to peer deps.

Reply:
They might :) This is a non-scientific assumption based on my personal experience while I've tested stuff.
We can probably sync work with @BoDonkey on that (I hope I'm mentioning the right Bob :) )